scripts/calendar.coffee

```diff
@@ -0,0 +1,152 @@
+# Description:
+#   Allows hubot to send Codio Calendar Events
+#
+# Dependencies:
+#   lodash google-auth-library googleapis ical
+#
+# Configuration:
+#   None
+#
+# Commands:
+#   calendar - show incoming events
+#
+# Notes:
+#   First time you run this script you must uncomment the
+#   "FIRST RUN" line below and exec this file alone.
+#   You will prompt to follow ad URL in Google and past a code.
+#   This is step is needed for authentication.
+#
+
+_ = require('lodash')
+moment = require('moment')
+
+Calendar = require('./lib/calendar')
+Utils = Calendar.utils
+
+TIME_CODIO_ENG_ADV = 60 * 15 * 1000  # 15 minutes
+TIME_HOLIDAY = 60 * 60 * 12 * 1000   # 12 hours
+TIME_SLEEP = 60 * 5000 # 5 mins
+
+ID_CODIO_ENG = 'codio.com_8fvaqs6pck1suoeqbv0nm0klno@group.calendar.google.com'
+ID_CODIO_HOL = 'applicationcraft.com_kl5hibe7h75sa0tt0uq11siiq8@group.calendar.google.com'
```

Comment:
This is a public repo, please don't include any Codio specific data. Use ENV vars instead.